static/bidder-params/kubient.json

```diff
@@ -3,5 +3,12 @@
   "title": "Kubient Adapter Params",
   "description": "A schema which validates params accepted by the Kubient adapter",
   "type": "object",
-  "properties": {  }
+  "properties": {
+    "zoneid": {
+      "type": "string",
+      "description": "Zone ID identifies Kubient placement ID.",
+      "minLength": 1
+    }
+  },
+  "required": ["zoneid"]
```

Comment:
Be careful here. I'm stealing this warning from @hhhjort:

> Do you have any current traffic coming from PBS? When you set zoneid as required, if a request comes in missing zoneid once it is set to required, it will mark the whole request as failed and not send a bid request to any bidder. So adding a required field can break the entire prebid setup of a publisher.
>
> This behavior is to help ensure publishers set up their configs correctly. Otherwise a bidder might set up a request with 5 bidders, but bidder ABC has a bad config and always errors out. If the other 4 are working normally, the publisher might not notice they have a bad config for quite a while. If some pages have a valid ABC config, they might just conclude that ABC just doesn't bid much if they don't troubleshoot carefully.
>
> If there are publishers who currently have configs that omit zoneid. we would recommend that initially the adapter not advertise zoneid as required in bidder-params, but return an error from within the adapter if it is missing. An error from within the adapter will not invalid the whole request, allowing the other bidders to function. Then after all publishers have had a reasonable amount of time to update their configs, we can do the additional step of making it required in bidder-params.

Reply:
Currently we don't have any traffic from prebid.
I'll change JSON schema to make **zoneid** as optional.

Reply:
Feel free at some later point (a few months) to submit a PR to make it required. I have less sympathy for a publisher who has been ignoring months of warning messages and the adapter not working. Just want to be sure publishers have sufficient warning before making a breaking change.